scripts/rest_examples/reservation-create-azure.http

```diff
@@ -5,7 +5,7 @@ X-Rh-Identity: {{identity}}
 
 {
   "name": "azure-linux-us-east",
-  "source_id": "5",
+  "source_id": "{{azure-source-id}}",
```

Comment:
This needs to be added in the `http-client-env.json` file I think GoLand will error out once this is used. But it’s fine.

Reply:
I've got it defined locally, there is no good default value for the comitted context IMO :thinking: